app/views/components/gds/exportsDateInput.scala.html

```diff
@@ -36,15 +42,30 @@
   if(errors.nonEmpty) Some(ErrorMessage(content = HtmlContent(errors))) else None
 }
 
+@bodyOrFieldsetTuple = @{bodyHtml
```

Comment:
tbh it's not really clear to me what's going on here. Maybe a simplification by using helper methods whose names could indicate what they are for? Just my 2 cents

Reply:
I've modified the template to use two options rather than a tuple (which does add to the complexity I agree). Any better?

Reply:
I've edited the template to use two options rather than a tuple (that does make things more complex I agree). Is this improvement any better?